vue-frontend/src/utils.js

```diff
@@ -30,6 +66,13 @@ export function analyticsEvent(refs) {
           return;
         }
       });
+    } else if (key == "portfoliolist") {
```

Comment:
Logic is same as `key == "portfolios"`. Isn't it better to use `OR` condition instead of duplicating the code?